docker/redis/server.crt

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
+MIIEzTCCArUCCQD3ffe5Oc6iWjANBgkqhkiG9w0BAQUFADAuMQ0wCwYDVQQKDARL
+b25nMR0wGwYDVQQDDBRLb25nIFRlc3RpbmcgUm9vdCBDQTAgFw0yMjA0MDgxMjQ3
+MDBaGA8zMDA3MTIwMTEyNDcwMFowITEfMB0GA1UEAwwWdGVzdC1yZWRpcy5leGFt
+cGxlLmNvbTCCAiIwDQYJKoZIhvcNAQEBBQADggIPADCCAgoCggIBALV/O7NdbDi9
+fyh3kwF+87CoQfeShx8+/xuZo3xopT1Ip+3z6m/30GS5Yk/M9ZBU25jYoYx1hTmG
+arlIPss8TExwxeBP6XP4wsXriYhI08r3q2uPRvG8kH07pIw3lmT6u0F5ivD7Io1L
+zDX0O2LUJyQI9/NV3+mQpJ38GgKiGt+yvEFdQ4Va47SYMWGkDZ0+oAhBTV6KRhTt
+JDJrbZe8uFNpipbUy3XfHc62W1sOdkwsuWZWaGlfWWWuWLS6z5lOzM9rB7gpa1XK
+Nug2k4OB3cBT9xszpVryJ/l0ds4sUfvjz9erJLWdTlA1Xy7nXI6KuPhWrVsCE8YJ
+EnQF6k6IFUafDO7gbg93sHBK3UJo4ZzGxGraCqrC2hh8xe8QWUukXk4O4v/oejyi
+h9ThjyqL6m+Da33AsvOI0zhw+XgLCSWFTlrzECIn8oxX+C/sA+aU06CcMWmtfq0v
+oVd5p4Mlk+nCZjx9clrO951BgP8CD9R+fz0JtLqaB3aziS0L8fyZYlHHEGkvMjq1
+G8yOV6AX8fVkKJwsYrdQOmkVhJgGYG4Y23j5+Z4i7vczBVKkmrP1XC79nu5Bd69d
+SDRhK2n9yIqJc2+ZlyZF/Nd8CCswyyXvd4rPqfB8jYoKIjza6ee2NPwMYmeW/wEe
+PX0wa351kdOCZD+KOqpM4QhK881kaUcpAgMBAAEwDQYJKoZIhvcNAQEFBQADggIB
+ABMjfY9zXaO3pxtUBCTChGi65mIj7YvedYAEStvfwi8PL3xfhlOjNT6Zn1m55gaY
+r3JpckrWTyOaCzZwB5QD67kA2dxBwst87a9lx+phM1mD6RSsBZyGSnF2WO6wDDki
+bXkPvZfPrwja1Bc3wynSHqKMhFV0eVePswVGbxMm8RTLXoB/HO0QmCHpKAV8FkD8
+/bnjdeW6TvK/d+8dhJU6XRRvJBM2RKWWi0iaZYuvq+KZoGBhnMsh9gVNXGZBrkAx
+mq52WlXvgEiBvFtnYoaQSE1Oi4y7ZUhEyr+pG4x9BGLAn84cS6s4QeMRoCisbadx
+KRCPPLuxa3FHbeIkremGSDoYjfT4p5uL5o8q+1BZ4dKhc9JPXwaNYPQFidVno9YZ
+GuLocqePTSTfd/jeOoNZ14AR5bfkpm/65elt3pgJ7gWf0HsDM7+q8V4HN+ruOXvP
+UPft0Tk69AA029ueMm80S0sReCH7jIy0OeEmdKmpO2f5F431+TIrAuTku1RIaa4i
+fegDVhkf7H/d8NOjHVBfQugXjBLWohyO+x3y0+KQY5RqHrwCSfjfUOLtN4XWoDe7
+sZCUU/nqLyOhGc+FKEpz5v9Bm8eaGoR5kh+srMr5Y3OtXLXi/xd1+IE9a2gWpzpE
+643zPI3Q/EFpuG6DQ1e1Qcze0VVP8Cwj0eYyvBJDad+8
+-----END CERTIFICATE-----
```

Comment:
The CN `test-redis.example.com` should be changed to something like `kong_redis_ssl`; this is to stay consistent with hybrid CN naming.